File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/soma_vetores.py
N: int

N = int(input("Quantos valores vai ter cada vetor? "))

A: [int] = [0 for x in range(N)]
B: [int] = [0 for x in range(N)]
C: [int] = [0 for x in range(N)]

print("Digite os valores do vetor A:")
for i in range(N):
	A[i] = int(input())

print("Digite os valores do vetor B:")
for i in range(N):
	B[i] = int(input())

for i in range(N):
	C[i] = A[i] + B[i]

print("VETOR RESULTANTE:")
for i in range(N):
	print(C[i])

File: nelio-alves/algoritmo-e-logica-de-programacao/cpp/exercicios/soma_vetores.cpp
#include <iostream>

using namespace std;

int main() {
    int N;

    cout << "Quantos valores vai ter cada vetor? ";
    cin >> N;

    int A[N], B[N], C[N];

    cout << "Digite os valores do vetor A:" << endl;
    for (int i = 0; i < N; i++) {
        cin >> A[i];
    }

    cout << "Digite os valores do vetor B:" << endl;
    for (int i = 0; i < N; i++) {
        cin >> B[i];
    }

    for (int i = 0; i < N; i++) {
        C[i] = A[i] + B[i];
    }

    cout << "VETOR RESULTANTE:" << endl;
    for (int i = 0; i < N; i++) {
        cout << C[i] << endl;
    }

    return 0;
}
